packages/mobile_app/lib/components/userinfo.dart

```diff
@@ -31,7 +31,7 @@ class UserInfo extends StatelessWidget {
         Padding(
           padding: const EdgeInsets.only(top: 16.0),
           child: Text(
-            lang.birthdate,
+            lang.birthdate.capitalize(),
```

Comment:
Should we capitalize this text? I'm really not sure, trying to recall what we have at mockups